rpc/context.go

```diff
@@ -191,27 +192,23 @@ func (ctx *Context) GRPCDial(target string, opts ...grpc.DialOption) (*grpc.Clie
 			log.Infof(ctx.masterCtx, "dialing %s", target)
 		}
 		conn, err := grpc.DialContext(ctx.masterCtx, target, dialOpts...)
```

Comment:
change this back to:

`meta.conn, meta.err = grpc.DialContext(ctx.masterCtx, target, dialOpts...)`